src/services/ApiService.ts

```diff
@@ -71,12 +78,31 @@ class ApiService {
 			})
 	}
 
+	/**
+	 * Use the fetch API over HTTP to hit the check token endpoint on the server
+	 *
+	 * @returns Promise that resolves user object if successful; otherwise, Promise that resolves to empty object
+	 */
+	pingGetUsersEndpoint(): Promise<GetUsersEndpointResult[]> {
+		return fetch(`${this._apiUrl}/user`, {
+			cache: NO_CACHE,
+			method: HTTP_GET,
+		})
+			.then((resp) => resp.json())
+			.then((obj) => obj)
+			.catch((err) => {
+				this.handleError(err)
+
+				return Promise.resolve([])
+			})
+	}
+
 	/**
 	 * Use the fetch API over HTTP to hit the check token endpoint on the server
 	 *
 	 * @returns Promise that resolves to true if token was valid; otherwise, Promise that resolves to false
 	 */
-	pingTokenCheckEndpoint(): Promise<boolean> {
+	 pingTokenCheckEndpoint(): Promise<boolean> {
```

Comment:
Did you add this space or did formatting add it?

Reply:
Ran prittier, re commited